src/lwc/rd2EntryForm/__tests__/rd2EntryForm.test.js

```diff
@@ -133,6 +133,7 @@ describe('c-rd2-entry-form', () => {
             await flushPromises();
 
             const elevateWidget = controller.elevateWidget();
+            expect(controller.disableElevateButton()).toBeTruthy();
```

Comment:
These are great additions to the test! We'll have to set these conditions up as part of the design next time, I know this got a little out of hand.